### ISSUE TITLE
Synchronization is paused indefinitely

### ISSUE THREAD

**Issue by almartin-fing** [owncloud/client#11519]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

The owncloud icon in the Mac OS menu bar is displayed as paused. When clicking it, one of the menu items shows "Synchronization is paused" in grey and there is no way to change this status.
Owncloud client version: 5.2.1.13040
Mac OS 14.3.1 (23D60)

### Expected behavior

Synchronization should not be paused; I have never asked to do so.
Also, I expect to find some menu option to enable synchronization again.

### Steps to reproduce the issue

I do not know how I got here

### Client version number

ownCloud 5.2.1.13040 [979bb9](https://github.com/owncloud/client/commit/979bb990f214fbd1cccc5536c582d84e196d499f)
Libraries Qt 6.4.3, OpenSSL 3.1.2 1 Aug 2023
Using virtual files plugin: suffix
OS: macos-23.3.0
QPA: cocoa

**Comment by TheOneRing** [comment]
Could it be that you are affected by https://github.com/owncloud/client/issues/11411

**Comment by TheOneRing** [comment]
if so, when did you update to 5.2.1, when was the first time you discovered the issue?

**Comment by almartin-fing** [comment]
Yes, this was the issue. Thanks for your help.
I am not sure when I updated to 5.2.1.
The ownclud.app file is dated 1/10/2024
I found a backup of ownclowd.cfg, dated 1/10/2024, with clientVersion=5.2.0.12726 that was already missing the sync configuration.
I had to recover the configuration from a previous backup, dated 12/3/2023, with clientVersion=5.1.2.12600
(notice that backup dates seem inconsistent with client versions)

I think I remember asking for "remembering to update later" for some update during January.
